yoti_test.go

```diff
@@ -217,7 +215,8 @@ func TestYotiClient_ParseWithoutProfile_Success(t *testing.T) {
 			result = &httpResponse{
 				Success:    true,
 				StatusCode: 200,
-				Content:    `{"receipt":{"wrapped_receipt_key": "` + wrappedReceiptKey + `",` + otherPartyProfileContent + `"remember_me_id":"` + rememberMeID + `", "sharing_outcome":"SUCCESS"}}`}
+				Content: `{"receipt":{"wrapped_receipt_key": "` + wrappedReceiptKey + `",` +
```

Comment:
Has this been run through `goimports`? It looks like the alignment is off.

Reply:
I currently go through the steps on the [CONTRIBUTING.md](https://github.com/getyoti/yoti-go-sdk/blob/master/CONTRIBUTING.md) page, as well as doing `gometalinter`, so we're doing `go fmt`. Would you suggest replacing `go fmt` with `goimports`? 

In any case, this shouldn't have been missed by `go fmt`, doesn't seem to be changed when I run it now either. I'll investigate

Reply:
Yeah, I had a play too. Since `goimports` essentially wraps `go fmt` it has the same behaviour, wasn't aware of this edge case.

It would be worth some time in the future looking into using `goimports`, it does all that `go fmt` does, but also sanitises imports (e.g. removes unused ones to prevent build failures down the line).